### ISSUE TITLE
health-v2-atom-oauth-sample

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/google-api-java-client#80]
```
Which API and version (e.g. Google Calendar Data API version 2)?
Google Health Data API version 2

What format (e.g. JSON, Atom)?
JSON

What Authentation (e.g. OAuth, OAuth 2, Android, ClientLogin)?
ClientLogin

Java environment (e.g. Java 6, Android 2.2, App Engine 1.3.7)?
Java 6

External references, such as API reference guide?


Please provide any additional information below.

```

Original issue reported on code.google.com by `Mehmet.A...@gmail.com` on 21 Dec 2010 at 7:34

**Comment by GoogleCodeExporter** [comment]
```
Marking as low priority because we strongly prefer our samples to use OAuth 1 
because we don't want applications to ask for user's passwords.  If you are 
willing to change your request to OAuth 1 we'll mark this Priority-Medium.
```

Original comment by `yan...@google.com` on 21 Dec 2010 at 12:30
- Changed title: **health-v2-json-clientlogin-sample**
- Changed state: **Accepted**
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Ok, I changed my request to OAuth.
```

Original comment by `Mehmet.A...@gmail.com` on 21 Dec 2010 at 12:46

**Comment by GoogleCodeExporter** [comment]
```
Besides being cool... is there a specific reason you're asking for JSON?  
Unfortunately, Health currently only supports Atom.
```

Original comment by `sax...@google.com` on 21 Dec 2010 at 1:01

**Comment by GoogleCodeExporter** [comment]
```
I used JSON few times so I am familiar with it. But there is no problem, Atom 
can be possible. If Health supports JSON in the future, I will be satisfied.
```

Original comment by `Mehmet.A...@gmail.com` on 21 Dec 2010 at 1:11

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the sample request.  There is currently a bug with our XML parsing 
code which is currently blocking the use of Google Health.  We expect to have 
this fixed by the version 1.3 release.
```

Original comment by `yan...@google.com` on 21 Dec 2010 at 2:41
- Changed title: **health-v2-atom-oauth-sample**
- Added labels: **Priority-Medium**
- Removed labels: **Priority-Low**

**Comment by GoogleCodeExporter** [comment]
```
The Google Health API has been deprecated, and will be discontinued January 1, 
2012.

http://googleblog.blogspot.com/2011/06/update-on-google-health-and-google.html
```

Original comment by `sax...@google.com` on 27 Jun 2011 at 6:18
- Changed state: **WontFix**
